### ISSUE TITLE
[React-Native] Dark Mode

### ISSUE THREAD

**Issue by bezedache29** [L-Blondy/tw-colors#32]
Hello, I am on React-Native 0.73.2 and I am using `twrnc` for my styles. My application needs to handle dark mode, and I would like to simplify the task by using the Tailwind classes only once for both light and dark modes. What I mean is, I don't want to have to use `text-black dark:text-white` everywhere. Instead, I would like the black color in my theme to automatically switch to white in dark mode. This way, every time I use black, it becomes white in dark mode, and the same logic applies to all colors.

On my WebApp, I use `tw-colors`, and it works very well, but I am having trouble reproducing it in React-Native.

Does anyone have any ideas on how I could integrate this functionality into my project ?

Thank you in advance for your responses.

**Comment by L-Blondy** [comment]
Hi @bezedache29

So far the only way I know to use tailwind in a react-native project is via NativeWind

Unfortunately NativeWind has limited support for css variables. See this issue for more details https://github.com/L-Blondy/tw-colors/issues/26#issue-1926931738

Feel free to let me know if you know another way to setup tailwind with react-native, I'll take a look into it

**Comment by L-Blondy** [comment]
Hi @bezedache29
I'll have a look at `twrnc`, tw-colors need full css variables support in order to work

**Comment by L-Blondy** [comment]
I'll close this issue since it is not a tw-colors issue, basically out of scope. 

If you need some help to find an alternative for react native or to setup your project you might start a discussion here https://github.com/L-Blondy/tw-colors/discussions/new/choose
